src/md5/md5.cc

```diff
@@ -215,31 +199,32 @@ void MD5::finalize (){
 
   finalized=1;
 
+  return true;
 }
 
 
 
 
-MD5::MD5(FILE *file){
-
-	init(); // must be called by all constructors
-  update(file);
-  finalize ();
+MD5::MD5(const unsigned char* input, uint32_t input_length) {
+	init();  // must be called be all constructors
```

Comment:
Introduces the typo again.

Reply:
I had so much fun with interactive rebasing and editing, squashing, splitting commits, i guess some fun slipped through the cage, will fix this later